xapian-core/weight/tfidfweight.cc

```diff
@@ -23,6 +23,10 @@
 #include <config.h>
 
 #include "xapian/weight.h"
+#include "weightinternal.h"
+#include "wdf-norm-dispatch.h"
+#include "idf-norm-dispatch.h"
+#include "keyword.h"
```

Comment:
Again, this new block ought to be in sorted order (but keep `xapian/weight.h` first as that's the header which has the definitions for this source file - we include that first as a simple way to try to ensure that our headers include any other headers they need (it doesn't ensure this for headers without a corresponding `.cc` file though).

Reply:
Updated